horizons/__init__.py

```diff
@@ -17,7 +17,7 @@ class Angle:
     """
     Angle in radians.
     """
-    µasPerDegree: float = 60 * 60 * 1000 * 1000
+    µasPerDegree: float =  60 * 1000
```

Comment:
The name and conversion here definitely isn't right. We have that there are indeed:
```
60 * 60 * 1000 * 1000
```
microarcseconds per degree as per the original line.

By eliminating a factor of 60 (?) and a factor of 1000, I am confused what unit we get here.

If we had:
```
60 * 60 * 1000
```
then we would have milliarcseconds per degree.

By removing a factor of 60, though, I'm not sure what this is supposed to be: something in arcminutes instead of arcseconds. I think maybe "milliarcminutes," which isn't really a unit that is used?

We should maybe make this:
```
milliarcsecondsPerDegree = 60 * 60 * 1000
```
and then change all references from microarcseconds to milliarcseconds?

Reply:
Of course, this will require changes to propagate to the test cases... references to microarcseconds should be converted to milliarcseconds, and you may need to recalculate some of the values.

Is everything passing now? I'm a bit confused by what's going on.